DesktopUI2/DesktopUI2/ViewModels/StreamViewModel.cs

```diff
@@ -232,6 +227,55 @@ public List<ActivityViewModel> Activity
       private set => this.RaiseAndSetIfChanged(ref _activity, value);
     }
 
+    private List<ApplicationObjectViewModel> _report;
+    public List<ApplicationObjectViewModel> Report
+    {
+      get => _report;
+      private set
+      {
+        this.RaiseAndSetIfChanged(ref _report, value);
+        this.RaisePropertyChanged("FilteredReport");
+        this.RaisePropertyChanged("HasReportItems");
+        this.RaisePropertyChanged("Log");
+      }
+    }
+    public List<ApplicationObjectViewModel> FilteredReport
```

Comment:
TODO later maybe, use a `SourceCache` for filtering instead as it will be more performant on large collections, see: https://github.com/specklesystems/speckle-manager2/blob/main/Speckle.Manager/ViewModels/ConnectorsViewModel.cs#L106